### PR TITLE
New method to get storage properties

### PR DESCRIPTION
**Pull request by DmitriiBobreshev**
**Description**: 
-- Added new method to get storage account properties for a given storage account 
-- Added private method to send request

**Documentation changes required:** N

**Added unit tests:**N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by vmapetr — APPROVED** [review]

**Review by kirill-ivlev — APPROVED** [review]

File: common-npm-packages/azure-arm-rest-v2/azure-arm-storage.ts [hunk 1]
```diff
@@ -212,6 +212,45 @@ export class StorageAccounts {
...
+            throw new Error(tl.loc("StorageAccountCannotBeNull"));
+        }
+
+        let response = await this.sendRequest(
+            "GET",
+            "//subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{storageAccountName}",
+            {
+                '{resourceGroupName}': resourceGroupName,
+                '{storageAccountName}': storageAccountName
+            }
```
Comment (ozanaydinMSFT) [inline common-npm-packages/azure-arm-rest-v2/azure-arm-storage.ts:230]: Is there a reason why don't we pass `{subscriptionId}` in this call?

File: common-npm-packages/azure-arm-rest-v2/azure-arm-storage.ts [hunk 2]
```diff
@@ -212,6 +212,45 @@ export class StorageAccounts {
...
+        );
+        
+        if (response.statusCode == 200) {
+            return response.body;
+        }
+
+        response = await this.sendRequest(
```
Comment (ozanaydinMSFT) [inline common-npm-packages/azure-arm-rest-v2/azure-arm-storage.ts:237]: Maybe a small comment might be helpful for future developers as why we're calling this endpoint.
e.g. `// call for Microsoft.ClassicStorage`

File: common-npm-packages/azure-arm-rest-v2/azure-arm-storage.ts (hunk 1 above)
Comment (DmitriiBobreshev) [reply common-npm-packages/azure-arm-rest-v2/azure-arm-storage.ts:232]: Yes, it passes automatically using [AzureServiceClient](https://github.com/microsoft/azure-pipelines-tasks/blob/master/common-npm-packages/azure-arm-rest-v2/AzureServiceClient.ts#L17), added description for the method, thank!

File: common-npm-packages/azure-arm-rest-v2/azure-arm-storage.ts (hunk 2 above)
Comment (DmitriiBobreshev) [reply common-npm-packages/azure-arm-rest-v2/azure-arm-storage.ts:241]: Added comment and removed {subscriptionId} argument, thanks!

**Review by Pranshu-Negi — APPROVED** [review]

**Pull request merged**